### DISCUSSION TITLE
AoS 0.76 support

### DISCUSSION THREAD

**Discussion by feikname** [piqueserver/piqueserver#632] [Feature Requests]

**Comment by NotAFile** [comment]
What does this require? Didn't pyspades have .76 support? Is it even worth it to use .76 when nobody uses it? Should we not just build on .75 instead?

**Comment by feikname** [comment]
@NotAFile [PySnip](https://github.com/NateShoffner/PySnip) (in which piqueserver is based) has 2  releases available: 0.75 and 0.76. Furthemore, there are 7 different branches in the PySnip repository, including a `master` and a `protocol076`, which differ from one another. I'm not sure if 0.76 is supported by piqueserver currently, but the fact that `protocol076` branch of PySnip differs from its `master`, and we are based on that very same `master` branch, leads me to believe that 0.76 is currently not supported. 

Regarding 0.76 being worth it, I think not. As you said: almost no one uses it. That's why I included the issue in the `Someday` milestone, it's a just in case issue for when we have already solved the other issues and can work on nice extra features.

**Comment by noway** [comment]
Yeah 0.76 is a strange patch in AOS history. It's not very popular either

**Comment by NotAFile** [comment]
unfortunately not :/

As I said, since I think the plan is to slowly Embrace, Extend, Extinguish the original .75 protocol, I'm not sure adding .76 makes any sense at all.

**Comment by NotAFile** [comment]
I've changed my mind on this. I think 0.76 would make sense, especially since we need to figure out a way to deal with serveral versions anway.

Much of that work (dynamically handling packet numbers etc) needs to be done either way, and 0.76 support is a nice goal on the way I guess.

**Comment by NotAFile** [comment]
@iamgreaser made this script that condenses all of the changes required in one place: https://raw.githubusercontent.com/iamgreaser/pysnip/master/feature_server/scripts/malsa076.py

As suspected, most of these are things that are not currently cleanly switchable at runtime.
